README.md

```diff
@@ -21,5 +21,35 @@ International licence.
 The survey metadata that is being converted to STAC can be viewed and downloaded
 from the [LINZ Data Service](https://data.linz.govt.nz/layer/51002).
 
-For more details see:
-https://www.linz.govt.nz/about-linz/what-were-doing/projects/crown-aerial-film-archive-historical-imagery-scanning-project
+For more details see
+[The Crown Aerial Film Archive historical imagery scanning project](https://www.linz.govt.nz/about-linz/what-were-doing/projects/crown-aerial-film-archive-historical-imagery-scanning-project).
+
+## Development
+
+Prerequisites:
+
+- [Poetry](https://python-poetry.org/)
+- [NPM](https://www.npmjs.com/)
+
+Follow these steps to set up a development environment with pre-commit hooks to
+automatically verify the contents of files before committing:
+
+1. Install packages:
+
+   ```shell
+   npm install
+   poetry install
+   ```
+
+2. Enable pre-commit hooks:
+
+   ```shell
+   poetry run pre-commit install --overwrite
+   ```
+
+## [License](LICENSE)
+
+[![Creative Commons License](https://i.creativecommons.org/l/by/4.0/88x31.png)](https://creativecommons.org/licenses/by/4.0/)
```

Comment:
Oh was the wrong license file incorporated? This is just CC-BY 4.0 rather than CC-BY-SA 4.0.

Reply:
Yep, my bad! I got the wrong plaintext file. In any case, the CC website doesn't host plaintext versions of their own licenses (!), so I've removed that file and left just the readme links.

Reply:
https://creativecommons.org/licenses/by/4.0/legalcode.txt ?

Reply:
Oh, thanks! I tried googling for it, and the link at https://creativecommons.org/licenses/by/4.0/ is just called "license". :P